examples/00_tutorials/plot_decoding_tutorial.py

```diff
@@ -52,7 +52,7 @@
 from nilearn import plotting
 from nilearn.image import mean_img
 
-plotting.view_img(mean_img(fmri_filename), threshold=None)
+plotting.view_img(mean_img(fmri_filename, copy_header=True), threshold=None)
```

Comment:
I have added the `copy_header=True` to all examples and also in the codebase (the test warnings were mostly because of that).

But now I am not so sure about this because we are anyway going to make it the default in 0.13.0.

Let me know what you think @Remi-Gau, @bthirion.